src/controllers/dynakube/istio/client.go

```diff
@@ -0,0 +1,180 @@
+package istio
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/pkg/errors"
+	istiov1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
+	istioclientset "istio.io/client-go/pkg/clientset/versioned"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type ClientBuilder func(config *rest.Config, scheme *runtime.Scheme, namespace string) (*Client, error)
+
+// Client - an adapter for the external istioclientset library
+type Client struct {
+	IstioClient istioclientset.Interface
```

Comment:
```suggestion
	IstioClientset istioclientset.Interface
```
istioClient = istio.Client
istioClient.IstioClient -- sounds wired, wdyt?

Reply:
done